src/icons/fileIcons.ts

```diff
@@ -201,10 +201,10 @@ export const fileIcons: FileIcons = {
         },
         { name: 'exe', fileExtensions: ['exe', 'msi'] },
         { name: 'java', fileExtensions: ['java', 'jar', 'jsp'] },
-        { name: 'c', fileExtensions: ['c', 'm'] },
+        { name: 'c', fileExtensions: ['c', 'm', 'i', 'mi'] },
         { name: 'h', fileExtensions: ['h'] },
-        { name: 'cpp', fileExtensions: ['cc', 'cpp', 'mm', 'cxx'] },
-        { name: 'hpp', fileExtensions: ['hpp'] },
+        { name: 'cpp', fileExtensions: ['cc', 'cpp', 'cxx', 'C', 'c++', 'CPP', 'cp', 'mm', 'M', 'mii', 'ii'] },
```

Comment:
Could you please remove the capitalized versions from all changes you made here e.g. "CPP" because VS Code is case insensitive here. Additionally you've to remove the uppercase "C" from cpp file extensions. It overwrites the "c" file extensions from c so that files ending with "*.c" would have a cpp icon, too.

Reply:
Ah, right. I forgot about case insensitive filesystems there. I'll fix it up :)

Reply:
I left HPP, but can also remove it. Not aware of other conflicts that could have.

Reply:
Thank you, but please remove the "HPP", too. It's sufficient if there's a lowercase "hpp", then the icon is also visible for files ending with "HPP". 

Conflicts are a good point. There are some check scripts that check, if there're some conflicts in the icon configuration but they don't yet check the character upper / lower cases. That might be added in the near future (#494) 😉

Reply:
Will do. Did some further testing on my end, I wasn't aware that VSCode was thoroughly case-insensitive when it came to things like this (didn't connect the dots from your first statement). I assumed it would use the host filesystem's attributes. Bad assumption on my part.

Reply:
Thank you, now it's fine 👍